.github/workflows/black.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Could you enforce headers in CI/Workflows?

Reply:
I'm not sure what you mean but yaml files do support comments yes, and the workflow ran and passed no problems (check youself below)

Reply:
I meant. What if I create a new file with no headers? Will the lint workflow fail?

Reply:
Good idea. I believe it's possible and I do like it. Thanks.